static/app/components/dropdownMenu/index.tsx

```diff
@@ -193,9 +199,9 @@ function DropdownMenu({
       <DropdownButton
         size={size}
         isOpen={isOpen}
-        {...triggerProps}
-        {...overlayTriggerProps}
         {...buttonProps}
+        {...overlayTriggerProps}
+        {...triggerProps}
```

Comment:
Setting the props after allows for styles coming in from `triggerProps` to override those from `buttonProps`